x/testnet.py

```diff
@@ -0,0 +1,24 @@
+from algosdk.v2client.algod import AlgodClient
+from algosdk.kmd import KMDClient
+from algosdk.v2client.indexer import IndexerClient
+
+DEVNET_TOKEN = "a" * 64
+ALGOD_PORT = 60000
+KMD_PORT = 60001
+INDEXER_PORTS = range(59_996, 60_000)
+
+
+def get_algod(force_retry: bool = False) -> AlgodClient:
```

Comment:
@tzaffi Minor:  Following https://github.com/algorand/py-algorand-sdk/pull/298/commits/bad794f92942457f2a6e1f511a0a59a2247ba892, I think it's safe to remove the `force_retry` parameter.
```suggestion
def get_algod() -> AlgodClient:
```